lib/Artwork.php

```diff
@@ -227,12 +241,102 @@ public static function GetByUrlPath($artistUrlName, $artworkUrlName): ?Artwork{
 		return $result[0];
 	}
 
+	public static function Build(string $artistName, ?int $artistDeathYear, string $artworkName, ?int $completedYear,
+				     bool $completedYearIsCirca, ?string $artworkTags, ?int $publicationYear,
+				     ?string $publicationYearPage, ?string $copyrightPage, ?string $artworkPage,
+				     ?string $museumPage): Artwork{
+		/** @return array<ArtworkTag> */
+		function parseArtworkTags(?string $artworkTags): array{
```

Comment:
nit: I vote to move this to a top-level `protected static` function. I get that it's only called from `Build()`, but it makes the `Build()` function longer than necessary.